repositories.json

```
@@ -99,7 +99,8 @@
     {"version": 4, "development": "4-dev", "release": "master"}
   ],
   "kourou": [
-    {"version": 0, "development": "develop", "release": "master"}
+    {"version": 0, "development": "develop", "release": "master"},
+    {"version": 1, "development": "1-dev", "release": "master"}
```

Kourou version is not ready yet. I think we could use `1-dev` as development branch for now then later remove the version 0 support. What do you think?
```suggestion
    {"version": 0, "development": "1-dev", "release": "master"}
```